include/swift/AST/GenericSignature.h

```diff
@@ -128,11 +128,10 @@ class alignas(1 << TypeAlignInBits) GenericSignature final
   void getSubstitutionMap(ArrayRef<Substitution> args,
                           SubstitutionMap &subMap) const;
 
-  using LookupConformanceFn =
-      llvm::function_ref<auto(CanType dependentType,
-                              Type conformingReplacementType,
-                              ProtocolType *conformedProtocol)
-                         -> Optional<ProtocolConformanceRef>>;
+  using GenericFunction = auto(CanType canType, Type conformingReplacementType,
+    ProtocolType *conformedProtocol)
+    ->Optional<ProtocolConformanceRef>;
```

Comment:
Its not your change, but Im not sure I understand the point of the trailing return with the auto.  Also, can you clang-format that line?  Pretty sure that it will add a space between `->` and the return type.